entity-framework/core/extensions/index.md

```diff
@@ -58,6 +58,12 @@ IWAPI (Instant Web API) is a scaffolding engine for .NET Core that can automate
 
 [Website](https://instantwebapi.com/)
 
+### efmig
+
+efmig is a multi-platform GUI application that speeds up daily development when working with Entity Framework Core. It covers the most popular use cases such as migration code and script generation with simple, one-click interface.
```

Comment:
Please add a sentence about supported EF Core versions (like it is done for all other entries here)